### DISCUSSION TITLE
高中學歷也能赴美工作！多益800就能徵應台積電美國21廠技術員

### DISCUSSION THREAD

**Discussion by YuHengChen54** [1101-GS3073-C/Python-Community-GS3073-C#462] [General]
高中學歷也能赴美工作！多益800就能徵應台積電美國21廠技術員
不知道是不是真的，但如果在台灣靠高中學歷真的不好找工作
https://www.technice.com.tw/autosystem/1240/

**Comment by BingRong1217** [comment]
讓我也突然想去美國了呢！
